### DISCUSSION TITLE
Parameters subtitution like psql quetion

### DISCUSSION THREAD

**Discussion by migueldeblas** [dbeaver/dbeaver#10685] [Q&A] [answered]
#### System information: 
- Windows 10
- DBeaver 7.3.0 eclipse plugin

#### Connection specification:
- Postgresql 12.5

#### Describe your question:

I work with psql -v/--set/--variable options scripts

I need to use:
:"param" because postgresql has case sensitive objects name;

The method of use is described here:
https://www.postgresql.org/docs/12/app-psql.html#APP-PSQL-INTERPOLATION 

In short if I use:
psql -v myschema=SchemaOne -v code=mycode
select count(*) from :"myschema"."TableExample" where code = :'mycode';
...

This code works in pgsql but fails in dbeaver.
SQL Error [42601]: ERROR: syntax error at or near ":"
(If Ichange :"xxx" or :'xxx' for :xxxxxx works)

I see a dbeaver parameter configuration (? and :) but I don't know how to configure it.
Is it a bug, is it not implemented or am I doing something wrong?

**Comment by kseniiaguzeeva** [comment]
Please delete quotes from parameter names:
select count(*) from :myschema.TableExample where code = :mycode

You will be able to add them in the value field for the value after pressing **Execute** button.
![Screenshot 2020-12-14 at 18 39 04](https://user-images.githubusercontent.com/51405061/102101498-c12ee280-3e3b-11eb-8dd9-59cd5767d6a4.png)

Please ask to reopen the ticket if it is not your case.

**Comment by migueldeblas** [comment]
Yes, if I delete quotes works in dbeaver (my develop tool) but doesn't work in psql that is the run tool used in production.
In postgresql it is mandatory double quote for upper/capitalize object names.
Changing to ":myschema" also doesn't work.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Please try to use following syntax ${xxx}. This is what we can offer from DBeaver side.
